lib/textmeasure/markdown.go

```diff
@@ -78,9 +80,31 @@ func HeaderToFontSize(baseFontSize int, header string) int {
 	return 0
 }
 
+func preProcessMarkdown(m string) string {
+	// This is a simplified pattern; a more robust one would be complex
+	urlRegex := regexp.MustCompile(`\b((?:https?|ftp)://[^\s"']+&\S*)\b`)
```

Comment:
I think we want to add `href=` and `src=` as a condition in this regex. Leave the links that are just text (and not hyperlinked) alone.

Reply:
I didn't think of that, great catch!